Fleece/Mutable/HeapValue.hh

```diff
@@ -42,8 +49,9 @@ namespace fleece {
             static void retain(const Value *v);
             static void release(const Value *v);
 
-            void* operator new(size_t size)             {return ::operator new(size);}
-
+            void* operator new(size_t size)                 {return ::operator new(size);}
+            void operator delete(void* ptr)                 {::operator delete(ptr);}
+            void operator delete(void* ptr, size_t size)    {::operator delete(ptr, size);}
```

Comment:
This is breaking the GCC and Clang builds (see the Travis results.) I think `operator delete` with a `size_t` parameter may be a MSVC-specific thing.

Reply:
I noticed that too.  Actually I think it might be a c++14 thing...I'll have to keep checking about this because I have no idea what is expected (though I have a better idea now because of the x86 Windows bug research)